### ISSUE TITLE
Open Letter to Cosmos Community

### ISSUE THREAD

**Issue by loredanacirstea** [allinbits/announcements#1]
Regarding https://github.com/allinbits/announcements/blob/main/2023_03_24_legalaction.md
The Cosmos Community is not able to provide a justice system that is at least as fair as the US Justice System and therefore, the Cosmos Community has positioned itself against the seeking of Justice, through prop #793 outcome. Given this, we felt a duty to write the following:

# Open Letter to Cosmos Community

Given: 
https://www.mintscan.io/cosmos/proposals/791
"#791 Legal Defense Fund Proposal"
https://www.mintscan.io/cosmos/proposals/793
"#793 Request to Drop the Lawsuit: All in Bits, Inc. vs. Grace Yu"

We, the developers of the Mythos chain stand together for Justice. We expect Justice to be served by the appropriate institutions only.

This is a link with most official info about the case (if you find a better public source: we will update it):
https://acrobat.adobe.com/link/review?uri=urn:aaid:scds:US:c1dbdfeb-b252-3682-a87e-ce07665ca613
https://www.pacermonitor.com/public/case/48253705/All_In_Bits,_Inc_v_Yu

We consider prop #793 as:
1) a violation of Article 10 from the [Universal Declaration of Human Rights](https://www.un.org/en/about-us/universal-declaration-of-human-rights)
> Article 10
> Everyone is entitled in full equality to a fair and public hearing by an independent and impartial tribunal, in the determination of his rights and obligations and of any criminal charge against him.
2) an attempt at [Obstruction of Justice](https://en.wikipedia.org/wiki/Obstruction_of_justice) using the blockchain as an instrument. This act falls under Criminal Law in most developed countries and blockchain governance should not have allowed such acts in principle.

When we volunteered for the Evmos chain, we proposed the chain adopt this principle by:
https://web.archive.org/web/20230226192853/https://www.mintscan.io/evmos/proposals/99
Such a proposal should be passed by each Cosmos chain in order to have a moral and legal Governance.

The developers of the Mythos chain want Justice to be served for Grace Yu @gyunit_, Denis Fadeev @fadeev, and All in Bits @allinbits

[Original source](https://github.com/the-laurel/chain-proposals/blob/main/cosmos/OpenLetterCosmosCommunity.md)
[Cosmos Reddit](https://www.reddit.com/r/cosmosnetwork/comments/13381i3/open_letter_to_the_cosmos_community/)
[Twitter](https://twitter.com/lorecirstea/status/1652433849731760129)

**Comment by ctzurcanu** [comment]
I am a developer for Mythos chain. I also approve and sign this open letter.

**Comment by CrownPrinceMac** [comment]
Big Organizations targeting an individual for speaking out is a miscarriage of Justice and the Antithesis of Human Rights. Your whole Open Letter is a perversion to Justice, giving a misconstrued definition.

**Comment by pwnfoo** [comment]
This is the stupidest shit I've heard all day and I've been on crypto Twitter since morning.

I'm sure this prop violates Geneva convention somehow too?

**Comment by fadeev** [comment]
Prop 793 is holding the deposited ATOMs hostage and threatens to burn them (if the quorum isn't reached), so it most likely does violate the Geneva Convention 🫠

**Comment by faddat** [comment]
Hey guys, aren't you the author's of the most vetoed proposals ever on EVMOS?

Wait, you don't need to answer that, I am well aware that you are.

https://www.mintscan.io/evmos/proposals/93
https://www.mintscan.io/evmos/proposals/95
https://www.mintscan.io/evmos/proposals/98
https://www.mintscan.io/evmos/proposals/99
https://www.mintscan.io/evmos/proposals/100
https://www.mintscan.io/evmos/proposals/105
https://www.mintscan.io/evmos/proposals/106

We didn't do all this work to make Cosmos to end up in idiotic court battles. I'm not surprised that loredana in particular would support this measure.  After being given very large EVMOS grants, you guys threatened to sue them. You like that stuff.

We don't.

PS: come at me bro

**Comment by oct-sheldon** [comment]
Article 10 reads verbatim:

"Everyone is entitled in full equality to a fair and public hearing by an independent and impartial tribunal, in the determination of his rights and obligations and of any criminal charge against him."

-Jae has no "rights" as the owner of a business, only as an individual: AiB sued Grace Yu, not Jae (businesses don't have rights)
-a "fair and public hearing by an independent and impartial tribunal" is the antithesis of a frivolous lawsuit, where the basis of that lawsuit has already been invalidated; the case against Grace Yu is based on factual information being considered damaging to the business of AiB based on a contract. As I've repeatedly posted on Twitter, factual information being highlighted by a former employee is verifiable on Github thanks to multiple individuals forking the Gnoland documentation, and is irrelevant to comments made by a former employee - they are simply verifiable fact that Jae posted ridiculous propaganda on Github
-AiB onchain partners have diluted the "fair and public hearing" that we've been pushing on Twitter for the past month. Jae has never joined any discussion, no representative of AiB has made any comment, and in some cases you were welcomed to join directly. The idea that an onchain proposal pleading for the counterfunding to resist a SLAPP suit hurts your rights to fair trial is LAUGHABLE
-You are summoning Article 10 and ignoring Article 11, the right to be presumed innocent until proven guilty. Your arguments are narrow and overtly biased based, as you summon the words of "justice" while trying to mark your opponents as "human rights abusers" because they don't recognize your perversions of justice. Example fluffed by you and your colleagues: https://twitter.com/lorecirstea/status/1652433849731760129 
-In contesting this SLAPP lawsuit there has been no single instance that could be regarded as "obstruction of justice". Individuals including me on Twitter have shared documents, engaged in debate, asked many to consider reasoning with details, and provided information about unethical and unreasonable behaviors happening at AiB, which has been public information since at least 2020 when several news articles were produced describing an unproductive and unacceptable work environment at AiB. I challenge you to reply to this message with any verifiable information that shows an "obstruction of justice" since you have seemingly no basis for this comment, other than the fact that other individuals contest the legitimacy of the lawsuit levied by AiB, while you complain that AiB's rights' are being infringed (again, individuals have rights, businesses do not)

Examples of published ethical concerns from organizations under Jae: 
https://decrypt.co/19769/zaki-manian-cosmos-number-two-resigns
https://cryptobriefing.com/cosmos-dev-leaves-tendermint-cites-untenable-ceo-reason/

Seriously, all yall needed to do was have a reasonable discussion about this, instead you summon irrelevant articles and apply legal pressure in brute force, without explanation or detail, citing only a single word with no clear legal definition in any English-accepting jurisdiction. Jae even liked a commenter who was uncertain about the situation on Twitter, refusing to address my comments directly after I pleaded for arbitration and speedy resolution. 

Your justice is fascism; your lack of principles are overt and don't need my color to verify that your "legal" argument here is without basis. I will not shut up because you accuse me of terrible things that have nothing to do with contesting a frivolous lawsuit.

**Comment by pwnfoo** [comment]
> Hey guys, aren't you the author's of the most vetoed proposals ever on EVMOS?
> 
> 
> https://www.mintscan.io/evmos/proposals/106


Wow, they really tried to do math on governance 😂😂😂😂😂😂

**Comment by onofreicl** [comment]
I approve this letter and sign as a validator of  the Mythos chain.

**Comment by pwnfoo** [comment]
> I approve this letter and sign as a validator of the Mythos chain.

This isn't Mythos, your shitcoins have no power here.

**Comment by ctzurcanu** [comment]
@faddat

> Hey guys, aren't you the author's of the most vetoed proposals ever on EVMOS?
> 
> Wait, you don't need to answer that, I am well aware that you are.
> 
> https://www.mintscan.io/evmos/proposals/93 https://www.mintscan.io/evmos/proposals/95 https://www.mintscan.io/evmos/proposals/98 https://www.mintscan.io/evmos/proposals/99 https://www.mintscan.io/evmos/proposals/100 https://www.mintscan.io/evmos/proposals/105 https://www.mintscan.io/evmos/proposals/106

Yes. We proudly defend what is right regardless of popular opinion, even at the expense of some of our time, sanity, and money. Thank you for mentioning our virtuous work.

> We didn't do all this work to make Cosmos to end up in idiotic court battles. 

No. You would rather be the judge (or at least instigator) of what you think is just even when the matter is beyond your jurisdiction. And persuade others to do the same and break the law together.

> I'm not surprised that loredana in particular would support this measure. After being given very large EVMOS grants, you guys threatened to sue them. 

We await a link to proof that we were "being given very large EVMOS grants". As far as we have proven: we are volunteers with thousands of hours of effort (you mentioned https://www.mintscan.io/evmos/proposals/95 and https://www.mintscan.io/evmos/proposals/100) for Evmos and we have not been paid a cent. We welcome any challenge to this statement.
Be aware that we reserve the right to sue you if you cannot produce proof, according to this algo:
https://github.com/the-laurel/chain-proposals/blob/main/laurel/EffortEvaluation.md
"you guys threatened to sue them" - clarify what you mean by "them" in this context and give a link to proof of our alleged threats too.

**Comment by oct-sheldon** [comment]
> No. You would rather be the judge (or at least instigator) of what you think is just even when the matter is beyond your jurisdiction. And persuade others to do the same and break the law together.

None of us instigated anything: individuals who care about Cosmos as a whole are responding to a frivolous lawsuit, that is provably a waste of ecosystem resources, time and legal liability in the state of New York. You can't just summon "the law" and simultaneously summon "beyond your jurisdiction" as if you have the secret power to control all jurisprudence such that we should bow to your colorful statements. 

A commenter cited onchain data and made a statement about it, you didn't counter that statement, and instead summoned a virtue signal to describe yourself as virtuous; you're hurting yourself.

**Comment by ctzurcanu** [comment]
@allinbits, Grace Yu, @fadeev 

I apologize for making you witnesses for side topics such as those about our previous work for Cosmos or ad hominem to us.
It is not fair that you lose your time reading irrelevant content. Justice will be applied to your case from the appropriate authority.
Also: it is not fair for us to not answer the most obvious attacks on us and our volunteering for Cosmos.
Therefore: I will answer subsequent side topics (if I must) from another repo and only link my answers to this one.

**Comment by pwnfoo** [comment]
>  Therefore: I will answer subsequent side topics (if I must) from another repo and only link my answers to this one.

If you're so sure of your PoV, why don't you come onchain and deposit your money to see what ATOM community thinks.

Come to the Cosmos Hub forum, discuss and start an on-chain proposal. Apparently you guys don't mind losing money on the countless veto you have got on other chains

**Comment by jaekwon** [comment]
closing because we dropped the suit, no longer needed.
relevant: https://github.com/allinbits/announcements/blob/main/2023_07_10_dropsuit_carmel.md

**Comment by loredanacirstea** [comment]
I think Project Carmel Is the right way for Cosmos and web3, if done fairly.
I think AiB should have renounced the lawsuit only after Project Carmel is completed AND a fair mediator is chosen for your particular case, with the pledge of all parties that they will respect the decision.

The mainstream legal system must remain a viable option if the parties do not agree on a process under a decentralized arbitration system.
But once parties accept such a process, the decision and the means for appeal must be respected, regardless of the outcome.
Good luck to all involved.

**Comment by oct-sheldon** [comment]
> I think Project Carmel Is the right way for Cosmos and web3, if done fairly. I think AiB should have renounced the lawsuit only after Project Carmel is completed AND a fair mediator is chosen for your particular case, with the pledge of all parties that they will respect the decision.
> 
> The mainstream legal system must remain a viable option if the parties do not agree on a process under a decentralized arbitration system. But once parties accept such a process, the decision and the means for appeal must be respected, regardless of the outcome. Good luck to all involved.

What is Project Carmel? Any published reference for the uninitiated?

**Comment by ctzurcanu** [comment]
The institution of "jubilee year" is described in Tanakh in [Leviticus 25:8-16](https://www.sefaria.org/Leviticus.25.8?lang=bi&with=all&lang2=en), 28-34, 39-41, 50-54.

"[...]and you shall hallow the fiftieth year. You shall proclaim release throughout the land for all its inhabitants. It shall be a jubilee for you [...]"

The jubilee year comes after 7x7 = 49 years of keeping the law. As the sabbath is the 7th day (the last day of the week period), the same way the jubilee year is after 49 years of normal years.
We have not seen 49 years of justice since the creation of decentralized finance and even less so since Cosmos. The community has not earned a jubilee year.

If your initial claim was just: you should not have yielded to the pressure of public opinion. Doing so will only encourage the bold abusers and the opinion-setters to become abusers. Plus: dropping the suit will make even those who initially believed you were a victim to doubt your moral standing. More so your moral fortitude.

By not submitting the case to the judgment of the authorities you have disregarded the very concept of Justice. You have not accepted the lawful mediator (and, applying to this the protocols of your proposed [Carmel Project](https://github.com/allinbits/carmel), the parties will not have agreement on a mediator (can't wait to be proven wrong on this)).
